src/SharpCode/Extensions.cs

```diff
@@ -46,6 +48,27 @@ public static string ToSourceCode(this Parameter parameter)
                 .Replace("{name}", parameter.Name);
         }
 
+        private static string DocumentationBlock(Option<string> documentation)
+        {
+            var docs = documentation.ValueOr(string.Empty);
+
+            if (string.IsNullOrWhiteSpace(docs))
+            {
+                return string.Empty;
+            }
+             
+            var blockParts = new[]
+            {
+                 "/// <summary>",
+                 "/// " + docs.Replace("\r\n", "\r\n/// ").Replace("\n", "\r\n/// ") ,
```

Comment:
Can you please use `\n` (unix style) as a new line character to be consistent with the rest of the methods? Same goes for line `@68` - `return string.Join(...`

You can also shorten this to a single `.Replace()` by using a RegEx like `/\r?\n/g` to match all new lines in replace them all at once.

Reply:
Fixed